app/setting/members/page.tsx

```diff
@@ -38,11 +38,14 @@ const Members = () => {
   }, []);
 
   const handleClick = async (hoveredItem: string | null, userId: string) => {
-    if (hoveredItem === 'Make Admin') {
+    if (hoveredItem === 'Make Admin' || hoveredItem === 'Remove Admin') {
       try {
         workspaceStore.setLoading(true);
         if (userId) {
-          const result = await makeAdmin({ userId, role: UserRole.ADMIN });
+          const result =
```

Comment:
Use like this to call same service :

const result = await updateRole({ userId, role: hoveredItem === 'Make Admin' ?  UserRole.ADMIN : UserRole.MEMBER })

Reply:
Done